packages/odata-common/src/entity-builder.ts

```diff
@@ -87,7 +87,7 @@ export class EntityBuilder<
 
   /**
    * Builds an entity from JSON representation.
-   * If you have obtained the JSON as a request payload use the {@link deserializeEntity} methods.
+   * If you have obtained the JSON as a request payload use the {@link entityDeserializer} methods.
```

Comment:
[pp] I think without object this doesn't make sense.

```suggestion
   * If you have obtained the JSON object as a request payload use the {@link entityDeserializer} methods.
```